go.mod

```diff
@@ -8,10 +8,11 @@ replace github.com/ethereum/go-ethereum => ./go-ethereum
 
 replace github.com/offchainlabs/bold => ./bold
 
+replace github.com/EspressoSystems/espresso-network-go => ./espresso-network-go
```

Comment:
I can't see the submodule in this PR. But go seems happy with it. That's confusing...